open-machine-learning-jupyter-book/llm/basic/attention.ipynb

```diff
@@ -0,0 +1,1598 @@
+{
```

Comment:
<p>Remove the redundant Enter key</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/735/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>